content.js

```diff
@@ -20,7 +24,21 @@ function toggleFiles() {
 
 	for (const el of rows) {
 		if (el.querySelector('.content a[title^="."')) {
```

Comment:
It should now be possible to not hide dotfiles. Everything should be go through the regex.